sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/ParallelDocumentQueryExecutionContextBase.java

```diff
@@ -142,7 +142,7 @@ protected void initializeReadMany(
                 return this.createDocumentServiceRequestWithFeedRange(headers,
                     querySpec,
                     null,
-                    feedRangeEpk,
+                    partitionKeyRange,
```

Comment:
seems this bug is introduced when we moved from partitionKeyRangeId to FeedRange.

I think this codepath is used both by query and readMany.

does this mean there is a partition split bug in the query as well?

Reply:
Query is fine https://github.com/Azure/azure-sdk-for-java/blob/4ca9e1915217f64ec96e0a1dfc92294f99588814/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/ParallelDocumentQueryExecutionContextBase.java#L81

Reply:
readmany uses a different initializer than query. So this would not effect query